pyproject.toml

```diff
@@ -53,6 +59,86 @@ warn_no_return = "False"
 module = [
     "pytorch_lightning.callbacks.progress.rich_progress",
     "pytorch_lightning.trainer.trainer",
-    "pytorch_lightning.tuner.batch_size_scaling"
+    "pytorch_lightning.tuner.batch_size_scaling",
+    "lightning_app.api.http_methods",
```

Comment:
That's a long list 😅 
And it took us 1y+ to get the ones in PL figured out. 
Good luck to us 🤞

Reply:
And a huge help from the community lol

Reply:
we'll get there :+1: